app/javascript/packs/quickstarts/container.js

```diff
@@ -1,9 +1,14 @@
 // @flow
 
 import { QuickStartContainerWrapper as QuickStartContainer } from 'QuickStarts/components/QuickStartContainer'
+import { getActiveQuickstart } from 'QuickStarts/utils/progressTracker'
 import { safeFromJsonString } from 'utilities/json-utils'
 
 document.addEventListener('DOMContentLoaded', () => {
+  if (getActiveQuickstart == null) {
```

Comment:
I might be missing something, but can you invoke the method without parenthesis?

Shouldn't it be:

```javascript
if (getActiveQuickstart() == null) {
  return 
}
```

Or more concisely I'd probably write it like this:

```javascript
if (!getActiveQuickstart()) {
  return
}
```